testing/checkpolicy/APKBUILD

```diff
@@ -0,0 +1,30 @@
+# Maintainer: Tycho Andersen <tycho@docker.com>
+pkgname=checkpolicy
+pkgver=2.6
+pkgrel=0
+pkgdesc="SELinux policy checker"
+url="http://www.selinuxproject.org"
+arch="all"
+license="GPLv2"
+depends="flex-dev"
+depends_dev=""
+makedepends="$depends_dev libsepol-dev bison flex flex-dev"
```

Comment:
This package doesn’t have -dev subpackage, so `$depends_dev` is useless here. Please remove both declaration of `depends_dev` variable and its usage in `makedepends`.